pages/providers/vpn.html

```diff
@@ -142,8 +142,11 @@ <h3>Should I use a VPN?</h3>
       <p><strong>In most cases, VPNs do little to protect your privacy or enhance your security</strong>, unless paired with other changes.</p>
       <p>VPNs cannot encrypt data outside of the connection between your device and the VPN server. VPN providers can see and modify your traffic the same way your ISP could. And there is no way to verify a VPN provider's "no logging" policies in any way.</p>
       <h3>What if I need encryption?</h3>
-      <p>In most cases, your traffic is already encrypted! Over 98% of the top 3000 websites offer <strong>HTTPS</strong>, meaning your traffic is safe regardless of using a VPN. It is incredibly rare for applications that handle personal data to not support HTTPS in 2019, especially with services like Let's Encrypt offering free HTTPS certificates to any website operator.</p>
+      <p>In most cases, most of your traffic is already encrypted! Over 98% of the top 3000 websites offer <strong>HTTPS</strong>, meaning your non-DNS traffic is safe regardless of using a VPN. It is incredibly rare for applications that handle personal data to not support HTTPS in 2019, especially with services like Let's Encrypt offering free HTTPS certificates to any website operator.</p>
       <p>Even if a site you visit doesn't support HTTPS, a VPN will not protect you, because a VPN cannot magically encrypt the traffic between the VPN's servers and the website's servers. Installing an extension like <a href="https://www.eff.org/https-everywhere">HTTPS Everywhere</a> and making sure every site you visit uses HTTPS is far more helpful than using a VPN.</p>
+      <h4>Should I use <a href="/providers/dns/#icanndns">encrypted DNS</a> with a VPN?</h4>
```

Comment:
I am not entirely happy with this, but I welcome better suggestions. Otherwise I would say that it's good enough.

Reply:
Suggestion: I think we don't need to hyperlink in this title since we already link to that page within the body of text.
```suggestion
      <h4>Should I use encrypted DNS with a VPN?</h4>
```